filament/src/components/RenderableManager.cpp

```diff
@@ -202,6 +203,88 @@ RenderableManager::Builder& RenderableManager::Builder::enableSkinningBuffers(bo
     return *this;
 }
 
+RenderableManager::Builder& RenderableManager::Builder::boneIndicesAndWeights(size_t primitiveIndex,
+               math::float2 const* indicesAndWeights, size_t count, size_t offset) {
+    std::vector<Entry>& entries = mImpl->mEntries;
+
+    ASSERT_PRECONDITION(primitiveIndex < entries.size() && primitiveIndex >= 0,
```

Comment:
Generally we try to do all the validation in `build()`. These setter methods should never fail and can be `noexecpt`.
The caller knows that the only method that can fail is `build()`.

Is it possible to make that change?

Reply:
I have fixed it